cmd/kubetoken/kubetoken.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubetoken
+
+import (
+	"fmt"
+
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+)
+
+func KubeToken() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "kubetoken",
+		Short: "Gets a token to access the Kubernetes API with client authentication",
+	}
+
+	cmd.RunE = func(*cobra.Command, []string) error {
+		c, err := okteto.NewKubeTokenClient()
+		if err != nil {
+			return fmt.Errorf("failed to initialize the kubetoken client: %w", err)
+		}
+
```

Comment:
Should we check somewhere in this command if the context is an Okteto one?

Reply:
Yeah, I just checked and the code that was preventing it
https://github.com/okteto/okteto/blob/e235fccd17bab97aac415afe17eec0905cc2e4c6/pkg/okteto/kubetoken_client.go#L35-L38
Doesn't show an understandable error for the user. Will change